### PR TITLE
CASMINST-3134 1.0: DOCS: Postgres health check failed with one error message for gitea-vcs-postgres pod.

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
Add additional postgres error that can be ignore to the validate csm health md
Resolves
* CASMINST-3134 : DOCS: Postgres health check failed with one error message for gitea-vcs-postgres pod. (for csm-1.0)

### CODE REVIEW

File: operations/validate_csm_health.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Execute ncnPostgresHealthChecks script and analyze the output of each individual
             INFO: running post_bootstrap
             INFO: trying to bootstrap a new cluster
          ```
-         Errors reported prior to the lock status, such as **ERROR: get_cluster** or **ERROR: ObjectCache.run ProtocolError('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))** can be ignored.
+         Errors reported prior to the lock status, such as **ERROR: get_cluster** or **ERROR: ObjectCache.run ProtocolError('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))** or **ERROR: failed to update leader lock** can be ignored.
```
Comment (brantk-hpe) [inline operations/validate_csm_health.md:194]: Would be easier to follow if this was a bulleted list.

**Review by brantk-hpe — APPROVED** [review]

File: operations/validate_csm_health.md (hunk 1 above)
Comment (bklei) [reply operations/validate_csm_health.md:194]: agree with this comment

**Review by bklei — APPROVED** [review]

**Review by rnoska-hpe — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

**Comment by johren-hpe** [comment]
Don't forget PRs for 1.1, 1.2, and main as well.

File: operations/validate_csm_health.md (hunk 1 above)
Comment (kimjensen-hpe) [reply operations/validate_csm_health.md]: fixed

**Comment by kimjensen-hpe** [comment]
> Don't forget PRs for 1.1, 1.2, and main as well.

Done

**Pull request merged**